extendr-api/src/robj/into_robj.rs

```diff
@@ -710,6 +710,16 @@ impl From<Vec<Rstr>> for Robj {
     }
 }
 
+#[cfg(feature = "faer")]
+impl From<faer::Mat<f64>> for Robj {
+    /// Convert a faer Mat<f64> into Robj.
+    fn from(mat: faer::Mat<f64>) -> Self {
+        mat.col_chunks(1)
+            .flat_map(|c| c.row_chunks(1).map(|r| r.read(0, 0)))
+            .collect_robj()
+    }
+}
+
```

Comment:
Should we also have an `impl From<RMAtrix> for faer::Mat<F64>>`?

Reply:
I think this question is more of the API design and what to support, so I'm not sure how to answer. Currently I think this provides the most flexibility and it's easy to go from an `Robj` to `RMAtrix`, or even make the conversion on the R side.